xarchain/chain.json

```diff
@@ -0,0 +1,122 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "xarchain",
+  "chain_type": "cosmos",
+  "chain_id": "xarchain",
+  "website": "https://arcana.network/",
+  "pretty_name": "XarChain",
+  "status": "live",
+  "network_type": "testnet",
+  "bech32_prefix": "cosmos",
+  "daemon_name": "xarchaind",
+  "node_home": "$HOME/.xarchain",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "stxar",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0,
+        "average_gas_price": 0,
+        "high_gas_price": 0
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "stxar"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/arcana/xarchain",
+    "recommended_version": "v0.0.5-test-rc",
+    "compatible_versions": [ ],
+    "consensus": {
+      "type": "cometbft",
+      "version": "v0.38.15"
+    },
+    "binaries": {
+      "linux/amd64": "https://github.com/arcana-network/xarchain/releases/download/v0.0.5-test-rc/v0.0.5-test_linux_amd64.tar.gz:checksum=sha256:9ba4870c7aed7f107acc3ec9bb361d5b5424c0b782c251f426403771924040fc"
+    },
+    "genesis": {
+      "genesis_url": "https://cosmos01-dev.arcana.network:26650/genesis"
+    },
+    "sdk": {
+      "type": "cosmos",
+      "version": "v0.50.10",
+      "tag": "v0.50.10"
```

Comment:
in the versions file, this property (sdk::tag) is missing. The version properties must agree (or only be defined in the versions file)